src/styles/main.scss

```diff
@@ -1,3 +1,52 @@
-body {
-  margin: 0;
+@import "./variables";
+
+.page {
+  font-family: $font;
+  font-size: 30px;
+}
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  width: $cell-size * 7 + $gap-size * 6 + 20px;
+  padding: $translation-height 10px 10px;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid black;
+    list-style: none;
+    background-color: $cell-color;
+    transition: 500ms;
+
+    @for $n from 1 through 31 {
+      &:nth-child(#{$n})::before {
+        content: "#{$n}";
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Why cursor: pointer; only for :hover pseudoclass?

Reply:
Thank you for your high appraisal! I have moved the "cursor: pointer;" instruction to the parent class. Please advise what the task tips say...:

"On hovering over a cell:

- cursor should become pointer..."